akka-docs/src/test/scala/docs/stream/RateTransformationDocSpec.scala

```diff
@@ -16,15 +16,12 @@ import akka.testkit.{ AkkaSpec, TestLatch }
 
 class RateTransformationDocSpec extends AkkaSpec {
 
-  type Seq[+A] = immutable.Seq[A]
-  val Seq = immutable.Seq
-
   implicit val materializer = ActorMaterializer()
 
   "conflate should summarize" in {
     //#conflate-summarize
     val statsFlow = Flow[Double]
-      .conflateWithSeed(Seq(_))(_ :+ _)
+      .conflateWithSeed(immutable.Seq(_))(_ :+ _)
```

Comment:
As this are in the actual samples (and the alias/companion object ref is not), better show what it is immutable.